fabric-tag-extensions-v0/src/main/java/net/fabricmc/fabric/mixin/tag/extension/MixinTagBuilder.java

```diff
@@ -57,4 +60,14 @@ public void clearTagEntries() {
 		entries.clear();
 		fabric_clearCount++;
 	}
+
+	@Override
+	public void addOptional(Identifier id) {
+		add(new Tag.OptionalObjectEntry(id), "fabric-tag-extensions");
```

Comment:
Source should match the fabric api module name completely.

```suggestion
		add(new Tag.OptionalObjectEntry(id), "fabric-tag-extensions-v0");
```